services/convert/git_commit.go

```diff
@@ -198,8 +198,17 @@ func ToCommit(ctx context.Context, repo *repo_model.Repository, gitRepo *git.Rep
 		affectedFileList := make([]*api.CommitAffectedFiles, 0, len(fileStatus.Added)+len(fileStatus.Removed)+len(fileStatus.Modified))
 		for _, files := range [][]string{fileStatus.Added, fileStatus.Removed, fileStatus.Modified} {
 			for _, filename := range files {
+				var filestatus = "unknown"
+				if util.SliceContainsString(fileStatus.Added, filename) {
+					filestatus = "added"
+				} else if util.SliceContainsString(fileStatus.Removed, filename) {
+					filestatus = "removed"
+				} else if util.SliceContainsString(fileStatus.Modified, filename) {
+					filestatus = "modified"
+				}
```

Comment:
No need to use "SliceContainsString"

In the outer loop `for _, files := range [][]string{fileStatus.Added, fileStatus.Removed, fileStatus.Modified} `, you already know current file is either added, removed or modified.

Reply:
Ok I modifed the code accordingly.